src/telemetry/metrics.yaml

```diff
@@ -12,20 +12,581 @@ page:
+        type: boolean
+      breach_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      legacy_breach_count:
+        description: Legacy users of Monitor may already have breaches reported on their dashboard, without having run the new free scan.
+        type: quantity
+      broker_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      session_id:
+        description: A flow ID (or similar) that allows us to track “sessions” of the user experience within the product.
+        type: string
+      utm_campaign:
+        description: Attribution campaign.
+        type: string
+      utm_content:
+        description: Attribution content.
+        type: string
+      utm_medium:
+        description: Attribution medium.
+        type: string
+      utm_source:
+        description: Attribution source.
+        type: string
+
+popup:
+  view:
+    type: event
+    description: |
+      A view of a popup, generated from clicking a button on a page view.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/MNTOR-2399
+    data_reviews:
+      - TODO
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    extra_keys:
+      url:
+        description: The full URL of the page.
+        type: string
+      user_id:
+        description: Mozilla acounts user ID. Randomly-generated if not logged in.
+        type: string
+      dashboard_tab:
+        description: Either [action_needed, fixed] depending on where the user is in their dashboard.
+        type: string
+      legacy_user:
+        description: This date is used to direct users who signed up after data broker scanning was released to the welcome flow.
+        type: boolean
+      breach_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      legacy_breach_count:
+        description: Legacy users of Monitor may already have breaches reported on their dashboard, without having run the new free scan.
+        type: quantity
+      broker_count:
+        description: Users who run the new free scan will be notified of breaches found during the scan.
+        type: quantity
+      session_id:
+        description: A flow ID (or similar) that allows us to track “sessions” of the user experience within the product.
+        type: string
+      utm_campaign:
+        description: Attribution campaign.
+        type: string
+      utm_content:
+        description: Attribution content.
+        type: string
+      utm_medium:
+        description: Attribution medium.
+        type: string
+      utm_source:
+        description: Attribution source.
+        type: string
+
+  exit:
+    type: event
+    description: |
+      An exit of a popup.
```

Comment:
Not sure if this is missing a word.

Reply:
Yes I agree it's a bit awkward, it's intended to fire when the user dismisses a popup.